scalding-core/src/test/scala/com/twitter/scalding/macros/MacrosUnitTests.scala

```diff
@@ -76,6 +99,16 @@ class MacrosUnitTests extends WordSpec with Matchers {
 
   def canExternalize(t: AnyRef) { Externalizer(t).javaWorks shouldBe true }
 
+  "MacroGenerated TupleConverter" should {
+    "Not compile for Option[Option[Int]]" in {
+      //TODO figure out a way to test this does not compile. See:
```

Comment:
What you can do is look for an implicit in a method with a default -- see the tests in bijection macros. You can assert that the implicit is the dummy implicit such that the implicit wouldn't work.